CONTRIBUTING.md

```diff
@@ -31,3 +31,152 @@ All submissions, including submissions by project members, require review. We
 use GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
+
+# Cross-platform Development
```

Comment:
I'm thinking about whether `COMTRIBUTING.md` is the right place to put these instructions. Maybe `devdoc.md` or `devdoc/environment.md` could be a better place?

Reply:
That seems like a good place for someone looking to contribute. I was inspired by Ktor: https://github.com/ktorio/ktor/blob/main/CONTRIBUTING.md

It certainly more discoverable and standard than devdoc.